__tests__/app.test.js

```diff
@@ -43,29 +43,29 @@ describe('News app', () => {
 
   describe("4. GET /api/articles/:article_id", () => {
     it("check the properties of an article object", () => {
-      let ARTICLE_ID = 2;
+      // let ARTICLE_ID = 2;
       return request(app)
-        .get(`/api/articles/${ARTICLE_ID}`)
+        .get(`/api/articles/2`)
         .expect(200)
         .then(({body:{ article }}) => {
           expect(article).toHaveProperty("author");
           expect(article).toHaveProperty("title");
-          expect(article).toHaveProperty("article_id");
```

Comment:
You should also check that the article_id is the same as the passed in `ARTICLE _ID`

Reply:
Fixed

Reply:
Still needs amending. 
We want to make a check to see that the article_id is actually 2 but at the moment you only check it has the key of article_id - add an assertion in this test to check the value of article_id from what comes back

Reply:
Fixed

Reply:
Hi Jeeann 👋🏽 Line 55 is checking the article_id YOU defined on line 44 is number 2 but we want you to check that `article_id` coming back from `body.article` is number 2 instead. Please look at that again, thanks!

Reply:
Perfect - line 56 is exactly what I mean 🎉